main.cpp

```diff
@@ -172,38 +195,54 @@ struct ButtonFactory
         std::string getButtonColor( std::string sku );
     };
 };
-/*
-Thing 2) Alien detection agency
-5 properties:
-    1) number of aliens on ground
-    2) fleet size
-    3) intelligence (IQ)
-    4) global coordinates detected from
-    5) case number
-3 things it can do:
-    1) process reports by field agents
-    2) filter out false reports
-    3) publish vetted reports
- */
+
+
+void ButtonFactory::makeButton(std::string sku, int amount)
+{
+    Button button;
+    button.skuInfo = sku;
+    amount += button.stockQuantity;
+}
+
+void ButtonFactory::designButton()
+{
+
+}
+
+void ButtonFactory::shipButtons(std::string address, int buttonType, int numToShip)
+{
+    std::cout << std::endl << "Ship To: " << address << std::endl << "Type: " << buttonType << std::endl << "Quantity: " << numToShip << std::endl;
+}
+
+void ButtonFactory::Button::orderButton(std::string sku, int amount)
+{   
+    amount -= stockQuantity;
+    skuInfo = sku;
+}
```

Comment:
Why are you subtracting `stockQuantity` from `amount`?   `stockQuantity` is a member variable.  `amount` is a function argument, which assumably means "I want this many buttons". 
Even though this project makes you write code for the sake of writing code, it's important to still write code that **makes sense.**
- [ ] revise